lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt

```diff
@@ -93,7 +93,7 @@ pyperclip==1.6.4
 pysam==0.14.1
 python-cinderclient==2.0.1
 python-dateutil==2.7.3
-python-genomespaceclient==0.2
+python-genomespaceclient==1.0.0
```

Comment:
Not a blocker for this PR, but can you do a 1.0.1 release and push the universal wheel on PyPI? That would save us to build it on Python 2.7 (see https://travis-ci.org/galaxyproject/galaxy/jobs/424522196#L905 ).

Reply:
There's a wheel on PyPI, but only for Python 3 (because it predates https://github.com/gvlproject/python-genomespaceclient/pull/8 ).